sidecar.sh

```diff
@@ -139,6 +164,11 @@ function main() {
         exit 1
     fi
 
+    if [[ -z "${MANIFEST_SERVICE_ENDPOINT}" ]]; then
```

Comment:
I don't think this section is needed

Reply:
You're right, thank you for catching this, I have removed this code in the latest commit